src/display/canvas.js

```diff
@@ -606,14 +624,17 @@ function compileType3Glyph(imgData) {
     c.scale(1 / width, -1 / height);
     c.translate(0, -height);
     c.beginPath();
```

Comment:
is it useful in Path2D case ?

Reply:
All ref-tests pass with it moved into the non-Path2D case, so I suppose it might be be needed unconditionally; however I don't *know* if it may somehow matter?